server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala

```diff
@@ -359,28 +360,29 @@ class DeltaSharingService(serverConfig: ServerConfig) {
   @Get("/shares/{share}/schemas/{schema}/tables/{table}/changes")
   @ConsumesJson
   def listCdfFiles(
-      @Header("User-Agent") @Nullable userAgent: String,
       @Param("share") share: String,
       @Param("schema") schema: String,
       @Param("table") table: String,
       @Param("startingVersion") @Nullable startingVersion: String,
       @Param("endingVersion") @Nullable endingVersion: String,
       @Param("startingTimestamp") @Nullable startingTimestamp: String,
-      @Param("endingTimestamp") @Nullable endingTimestamp: String): HttpResponse = processRequest {
+      @Param("endingTimestamp") @Nullable endingTimestamp: String,
+      @Param("returnMetadata") @Nullable returnMetadata: String): HttpResponse = processRequest {
```

Comment:
should we call this returnMetadataChanges since queryTableChanges already returns the latest metadata ?

Reply:
I don't know. 
I'm not fully happy with returnMetadata either. But returnMetadataChanges sounds like the changes of metadata, which we are returning is just the metadata it self. They are the additional metadata seen in the deltaLog. 
returnAdditionalMetadata? not as good as returnMetadata even..